WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -519,6 +492,46 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
     }
 }
 
+// MARK: - Quick Start Prompt
+private extension WordPressAuthenticationManager {
+    func presentQuickStartPrompt(for blog: Blog, in navigationController: UINavigationController, onDismiss: ((Blog, Bool) -> Void)?) {
+        // If the quick start prompt has already been dismissed,
+        // then show the My Site screen for the specified blog
+        guard !quickStartSettings.promptWasDismissed(for: blog) else {
+
+            if self.windowManager.isShowingFullscreenSignIn {
+                self.windowManager.dismissFullscreenSignIn(blogToShow: blog)
+            } else {
+                navigationController.dismiss(animated: true)
+            }
```

Comment:
I _think_ we can get rid of the if-else here, now that we're explicitly handling adding self-hosted sites.

```suggestion
            self.windowManager.dismissFullscreenSignIn(blogToShow: blog)
```

Reply:
We can't! There's one single flow that will break if we do this change:

1. Login into an account with no sites
2. Tap "Add a self-hosted site"
3. Fill the fields
4. On the end, you'll get stuck on the last screen (it will never be dismissed)